extension/storage/Makefile

```diff
@@ -0,0 +1 @@
+include ../../Makefile.Common
```

Comment:
Why this?

Reply:
Artifact from being a contrib component. Should be removed in core implementation.